README.md

```diff
@@ -148,6 +163,20 @@ pip install mbodied[audio]
 
 ## Getting Started
 
+### Customize a Motion to fit a robot's action space.
```

Comment:
This should go below notebooks. The first item in getting started should be the most important thing.